evennia/server/evennia_launcher.py

```diff
@@ -732,8 +732,12 @@ def kill(pidfile, signal=SIG, succmsg="", errmsg="",
                   "The PID file 'server/%(pidfile)s' seems stale. "\
                   "Try removing it." % {'pid': pid, 'pidfile': pidfile})
             return
-        print("Evennia:", succmsg)
-        return
+        try:
```

Comment:
I think it is a dangerous assumption to put the try statement here since I suspect it's time dependent from machine to machine. What happens is that the SIGINT(-like) signal being sent will hit the launcher (since Windows can only send SIGINT to *all* of the Console's processes at the same time, yeah - that's a thing). But exactly where in the source the execution happens when the SIGINT hits is time dependent. In your case the computer happens to be at the print statement; this may not be the case for others. So if a try...except is needed in this module I fear it must encompass all of this section. Or the Windows version should, at least.